src/hotspot/os/windows/perfMemory_windows.cpp

```diff
@@ -222,7 +222,7 @@ static bool is_directory_secure(const char* path) {
     else {
       // unexpected error, declare the path insecure
       if (PrintMiscellaneous && Verbose) {
-        warning("could not get attributes for file %s: ",
```

Comment:
Too bad ATTRIBUTE_PRINTF is empty for Windows.  Visual Studio doesn't seem to be an equivalent to
gcc's `__attribute__((format(printf(...)))`.

Reply:
The best I could find in Visual Studio was [\_Printf_format_string\_](https://learn.microsoft.com/en-us/cpp/code-quality/annotating-function-parameters-and-return-values?view=msvc-170#format-string-parameters)
It needs to be specified next to the parameter, and it's only checked when compiling with `-analyze`, so not really useful for us at this moment.